incubator/binding-risingwave.spec/src/main/scripts/io/aklivity/zilla/specs/binding/risingwave/streams/effective/create.function.return.struct/server.rpt

```diff
@@ -35,8 +35,8 @@ read zilla:data.ext ${pgsql:dataEx()
                                 .build()
                               .build()}
 read "CREATE FUNCTION key_value(bytea)\n"
-     "RETURNS struct < key varchar , value varchar >\n"
-     "AS key_value\n"
+     "RETURNS struct<key varchar, value varchar>\n"
```

Comment:
Same question here about spaces? Checking if they are no longer permitted or not.